pkg/testutils/lint/passes/norawcookies/testdata/src/a/a.go

```diff
@@ -0,0 +1,25 @@
+package a
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/cockroachdb/cockroach/pkg/util/httputil"
+)
+
+func init() {
+	_ = http.Cookie{ // want `raw cookie construction not permitted!`
+		Name: "shouldn't be allowed!",
+	}
```

Comment:
Out of pure curiosity, does this linter catch an aliased import?

Reply:
Nope, but I was able to add it.